cfme/tests/pod/test_appliance_crud.py

```diff
@@ -324,3 +342,59 @@ def test_aws_smartstate_pod():
         initialEstimate: None
     """
     pass
+
+
+def test_pod_appliance_db_backup_restore(temp_pod_appliance, provider, setup_provider,
+                                         template_folder):
+    """
+    This test does the following:
+      - adds openshift provider where openshift based appliance is situated
+      - creates pvc and db backup in it
+      - stops appliance
+      - restores db from snapshot
+      - starts appliance and finds that appliance in CloudForms
+    """
+    template_folder = template_folder
+    appliance = temp_pod_appliance
+    assert provider.mgmt.is_vm_running(appliance.project)
```

Comment:
**TODO**

Out of scope, but a thought to share - a PodAppliance subclassing Appliance could hold methods related to introspection of podified appliance - like if the project pod is running.

Reply:
I came up with such a proposal several times but @psav  and mfalesnik had another plan. They were planning to significantly change current appliance structure.

I can implement such class if you endorse this change.